### PR TITLE
Upgrade to Node 20 LTS

### PR DESCRIPTION
**Pull request by kaufmo**

### CODE REVIEW

**Review by dkarnutsch — APPROVED** [review]

**Comment by johnnyomair** [comment]
Please also update create-app/ (we should add this to install.sh)

**Comment by kaufmo** [comment]
> Please also update create-app/ (we should add this to install.sh)

I'm already on it :+1:

File: create-app/src/util/isValidNodeVersion.ts [hunk 1]
```diff
@@ -1,8 +1,8 @@
-function isNodeVersionGreaterThan18(): boolean {
+function isNodeVersionGreaterThan20(): boolean {
     const [major] = process.versions.node.split(".").map(Number);
-    return major >= 18;
+    return major >= 20;
```
Comment (johnnyomair) [inline create-app/src/util/isValidNodeVersion.ts:3]: Is this what we want? Or may we risk allowing Node 18?

Comment (dkarnutsch) [reply create-app/src/util/isValidNodeVersion.ts:3]: we could require new projects to start with nodejs 20

**Review by johnnyomair — APPROVED** [review]

**Review by thomasdax98 — APPROVED** [review]

**Review by dkarnutsch — APPROVED** [review]

**Pull request merged**